minds/datasources/datasources.py

```diff
@@ -16,11 +16,12 @@ class DatabaseConfig(BaseModel):
 class Datasource(DatabaseConfig):
     ...
 
+
 class Datasources:
     def __init__(self, client):
         self.api = client.api
 
-    def create(self, ds_config: DatabaseConfig, replace=False):
+    def create(self, ds_config: DatabaseConfig, replace=False, update=False):
```

Comment:
I don't like the word 'replace' here. Without code inspection i can't predict what would happen if set this option to 'true'. For example, if i have mind 'X' with attached datasource 'Y', and then call `datasource.create('Y', replace=True)`, then datasource 'Y' will be detached from mind 'X'. It does't look like 'replace'.
The presence of two options 'replace' and 'update' also make behavior undefined. What if i set both to 'true'?
Better will be to split this into two methods without 'replace' or 'update' arguments:
 - create - raise error if ds exists
 - upsert - update if exists, create otherwise

Reply:
I will remove 'replace' parameter